tests/ignite/handlers/test_checkpoint.py

```diff
@@ -28,6 +28,10 @@ def forward(self, x):
         return self.net(x)
 
 
+model = DummyModel()
```

Comment:
Actually, every test allocates locally a model. Having a global model is the most impacting change in this PR. Maybe it would be better to use a fixture rather than this global approch, what do you think ?

Reply:
@sdesrozis model object is the same for all tests and it is not reallocated I think. You can check that with the following check:
```python
class A:
    pass


model = A()

def test_1():
    print("test_1", id(model), model)
    assert False


def test_2():
    print("test_2", id(model), model)
    assert False
```

```
pytest tests/ignite/test_global_var.py

tests/ignite/test_global_var.py FF                                                                                                                                         [100%]

==================================================================================== FAILURES ====================================================================================
_____________________________________________________________________________________ test_1 _____________________________________________________________________________________

    def test_1():
        print("test_1", id(model), model)
>       assert False
E       assert False

tests/ignite/test_global_var.py:12: AssertionError
------------------------------------------------------------------------------ Captured stdout call ------------------------------------------------------------------------------
test_1 140154206763856 <tests.ignite.test_global_var.A object at 0x7f7831b53b50>
_____________________________________________________________________________________ test_2 _____________________________________________________________________________________

    def test_2():
        print("test_2", id(model), model)
>       assert False
E       assert False

tests/ignite/test_global_var.py:17: AssertionError
------------------------------------------------------------------------------ Captured stdout call ------------------------------------------------------------------------------
test_2 140154206763856 <tests.ignite.test_global_var.A object at 0x7f7831b53b50>
============================================================================ short test summary info =============================================================================
FAILED tests/ignite/test_global_var.py::test_1 - assert False
FAILED tests/ignite/test_global_var.py::test_2 - assert False
```